kubejs/server_scripts/farmerscompat.js

```diff
@@ -2,32 +2,26 @@ onEvent("recipes", (event) => {
     mixingRecipeGen(event);
 })
 
-var FULL_BUCKET_AMMOUNT = 81000;
-var INGOT_FLUID_AMMOUNT = 9000;
+const FULL_BUCKET_AMOUNT = 81000;
+const INGOT_FLUID_AMOUNT = 9000;
 
 function mixingRecipeGen(event) {
     event.forEachRecipe({type:"farmersdelight:cooking"}, recipe => {
         //Gets item name without namespace (e.g. "cooked_rice")
-        var outputItem = recipe.getOriginalRecipeResult().getItem().getId().substring(recipe.getOriginalRecipeResult().getItem().getId().indexOf(":")+1); 
+        var outputItem = recipe.getOriginalRecipeResult().getId().split(":")[1]; 
         var inputItems = recipe.getOriginalRecipeIngredients();
-        var container;
-        
-        //Determines container for spouting recipe
-        if(outputItem=="hot_cocoa" | outputItem=="apple_cider" | outputItem=="hamburger") {
-            container="minecraft:glass_bottle";
-        } else if(outputItem=="stuffed_pumpkin_block") {
-            container="minecraft:pumpkin";
-        } else if(outputItem=="dumplings") {
-            container="#c:dough";
-        } else {
-            container="minecraft:bowl";
+        let containers = {
+            "minecraft:glass_bottle": [ "hot_cocoa","apple_cider","hamburger"],
+            "minecraft:pumpkin": ["stuffed_pumpkin_block"]

```

Comment:
What I sent was just an example, and now you're missing dumplings! Good job adding the null case, I totally forgot about that